src/com/epam/javatrainings/io/DeleteFile.java

```diff
@@ -0,0 +1,31 @@
+package io;
+
+import java.io.File;
+
+public class DeleteFile {
+    public static void main(String[] args) {
+        File sourceDir = new File("C:\\Temp");
+        listFiles(sourceDir);
+    }
+    
+    private static void listFiles(File parentDir){
+        if(!parentDir.isDirectory()){
+            System.out.println("Not a directory.");
+            return;
+        }
+        File[] fileList = parentDir.listFiles();
```

Comment:
Use Java 8 Stream instead of this loop.
Use Files.walk() to iterate through files list